### PR TITLE
Added: Pajda, Rajskie Smaki, Green2go

### PR DESCRIPTION
**Pull request by PiotrekGlab**
In order to keep our vendors updated please consider merging.

### CODE REVIEW

File: scripts/mail2slack.py [hunk 1]
```diff
@@ -119,6 +119,9 @@ def processEmails(self):
 vendors.add(Vendor('Ślimak', ['slimak', 'Ślimak', 'ślimak'], 'Pan Ślimak'))
 vendors.add(Vendor('Jesz to', ['Jeż.to', 'Jeż', 'jeż.to', 'jeż', 'Jez.to', 'Jez', 'jez.to', 'jez'], 'Jesz to'))
 vendors.add(Vendor('Borówka', ['BORÓWKA', 'BOROWKA', 'borówka', 'borowka'], 'Pan Borówka'))
+vendors.add(Vendor('Pajda', ['Pajda', 'pajda',], 'Pajda'))
+vendors.add(Vendor('Rajskie Smaki', ['Rajskie Smaki', 'Rajskie smaki', 'rajskie smaki', 'ajskie Smaki'], 'Rajskie Smaki'))
```
Comment (moczarskim) [inline scripts/mail2slack.py]: I will not trust firm 'ajskie Smaki' ;)

Comment (PiotrekGlab) [reply scripts/mail2slack.py]: Typo fixed.

File: scripts/mail2slack.py [hunk 2]
```diff
@@ -119,6 +119,9 @@ def processEmails(self):
 vendors.add(Vendor('Ślimak', ['slimak', 'Ślimak', 'ślimak'], 'Pan Ślimak'))
 vendors.add(Vendor('Jesz to', ['Jeż.to', 'Jeż', 'jeż.to', 'jeż', 'Jez.to', 'Jez', 'jez.to', 'jez'], 'Jesz to'))
 vendors.add(Vendor('Borówka', ['BORÓWKA', 'BOROWKA', 'borówka', 'borowka'], 'Pan Borówka'))
+vendors.add(Vendor('Pajda', ['Pajda', 'pajda',], 'Pajda'))
+vendors.add(Vendor('Rajskie Smaki', ['Rajskie Smaki', 'Rajskie smaki', 'rajskie smaki', 'rajskie Smaki'], 'Rajskie Smaki'))
```
Comment (piaseckim) [inline scripts/mail2slack.py]: Woudn't "Rajskie" be just enough?

Comment (piaseckim) [reply scripts/mail2slack.py]: I meant in the keywords array for vendor identification

Comment (PiotrekGlab) [reply scripts/mail2slack.py]: it all depends whether we want to fix any similar naming when next vendor show up with "rajskie" in its name or we want to secure it now.
Your call about this one.

**Pull request merged**